path_planning_plugins/include/path_planning_base.h

```diff
@@ -0,0 +1,19 @@
+#ifndef PATH_PLANNING_BASE_H_
+#define PATH_PLANNING_BASE_H_
+
+#include <geometry_msgs/PoseArray.h>
+#include <godel_msgs/PathPlanningParameters.h>
+#include <pcl/PolygonMesh.h>
+
+namespace path_planning_base
+{
+  typedef  godel_msgs::PathPlanningParameters PlanningParams;
```

Comment:
If it's not in the public interface, we might as well move it to the source file.